tests/test_video.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import pytest
+
+import tensorflow as tf
+from tensorflow.keras.applications.resnet50 import ResNet50
+from tensorflow.keras.applications.resnet50 import preprocess_input, decode_predictions
+from tensorflow_io.video import VideoDataset
+
+video_path = os.path.abspath(
+    os.path.join(os.path.dirname(os.path.abspath(__file__)),
+    "..",
+    "tensorflow_io/video/python/kernel_tests/testdata/small.mp4"))
```

Comment:
Should we move all test data to top level since it's used by top level `tests/` as well?

Reply:
The only issue is the bazel as `testdata` directory needs to be visible to the project e.g., `//tensorflow_io/video/BUILD`, haven't found out a successful way to make it work with bazel yet.

I can continue investigation and add a follow up PR once I am able to expose the testdata in bazel.